.github/workflows/benchmarks.yml

```
@@ -0,0 +1,56 @@
+name: Benchmarks
+on:
+  schedule:
+    - cron: "0 0 * * 0-6"  # Run daily at 0am
+  pull_request:
+
+concurrency:
+  group: benchmarks-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+    # If needed, this workflow will create a cached benchmark reference.
+    # This reference will be used for comparative analyses with all PRs benchmarks.
+    reference-benchmarks:
+      if: github.event_name == 'schedule' || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'ci:run-benchmarks'))
+
+      uses: ./.github/workflows/interface-unit-tests.yml
+      with:
+        branch: add-pytest-benchmarks

```

Let's add a `TODO` here so we remember it needs to be updated.
```suggestion
        # TODO: Update this in the future to a stable release or master branch
        branch: add-pytest-benchmarks

```

---

This will change soon, but OK.